docs/get-started/features.md

```diff
@@ -606,7 +606,7 @@ feature.node.kubernetes.io/override_source-OVERRIDE_VALUE=123
 override.namespace/value=456
 ```
 
-#### A File Example
+#### A File example
```

Comment:
```suggestion
#### A file example
```

Reply:
Thanks, fixed! There was a similar omission on "A Hooks Example" that I also fixed